src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -292,27 +293,30 @@ private static AuditListener createListener(String format,
 
         // setup the output stream
         OutputStream out;
-        boolean closeOut;
+        boolean closeOutputStream;
         if (outputLocation != null) {
             out = new FileOutputStream(outputLocation);
-            closeOut = true;
+            closeOutputStream = true;
         }
         else {
             out = System.out;
-            closeOut = false;
+            closeOutputStream = false;
         }
 
         // setup a listener
         AuditListener listener;
         if ("xml".equals(format)) {
-            listener = new XMLLogger(out, closeOut);
+            listener = new XMLLogger(out, closeOutputStream);
 
         }
         else if ("plain".equals(format)) {
-            listener = new DefaultLogger(out, closeOut);
+            listener = new DefaultLogger(out, closeOutputStream);
 
         }
         else {
+            if (closeOutputStream) {
+                IOUtils.closeQuietly(out);
```

Comment:
@baratali, why can't we use just Guava? http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/io/Closeables.html

I don't think we want any additional dependency in Checkstyle.